aws_lambda_powertools/event_handler/content_types.py

```diff
@@ -0,0 +1,2 @@
+APPLICATION_JSON = "application/json"
+PLAIN_TEXT = "plain/text"
```

Comment:
This should be “text/plain”

Reply:
Thanks for catching it, fixed now in develop w/ a note on `mimetypes` lib.